core/object/script_language.h

```diff
@@ -428,6 +428,8 @@ class ScriptLanguage : public Object {
 	virtual int profiling_get_frame_data(ProfilingInfo *p_info_arr, int p_info_max) = 0;
 
 	virtual void frame();
+	virtual void set_physics_process_delta_time(double p_time);
+	virtual void set_process_delta_time(double p_time);
```

Comment:
I thought of using the `process_frame` and `physics_frame` signals, but our callback needs to have priority, as we want the delta time to be updated before the other callbacks are called.